### PR TITLE
Update standards

### PR DESCRIPTION
**Pull request by macumber**
@jmarrec i am just trying to hack this to get it to run right now

### CODE REVIEW

**Comment by macumber** [comment]
@jmarrec i re-exported 90_1_2013.osm, if you have time to see if it makes sense tomorrow i could look at running to export all the other standards too

**Comment by macumber** [comment]
Whoops, accidentally exported with 2.9.1, re-doing with 3.1.0-rc1 now

**Comment by macumber** [comment]
OK, updated with 3.1.0-rc1

**Comment by jmarrec** [comment]
@macumber In order to have a cleaner history, I first copied the standards script **as-is** then cherry picked your commits atop of it, so I could better the changes you made to it. I then **force-pushed** so make sure to update your refs

**Comment by jmarrec** [comment]
@macumber I copied the existing 90.1-2013.osm, and VT'ed it to 3.1.0-rc1. Then I copied yours.

I've ran a regular diff, but that's a bit hard to follow sometimes, so I made this small helper:

```
@m_ori = osload('90_1_2013_ori_vt.osm')
@m_mod = osload('90_1_2013_mod.osm')

@m_ori = osload('90_1_2013_ori_vt.osm')
@m_mod = osload('90_1_2013_mod.osm')

def compare(className, return_names=false)
  x_ori = eval("@m_ori.get#{className}s.map{|x| x.nameString}")
  x_mod = eval("@m_mod.get#{className}s.map{|x| x.nameString}")
  deleted = x_ori - x_mod
  if !deleted.empty?
    puts "Deleted #{className}:"
    puts deleted
    puts "\n\n"
  end
  added = x_mod - x_ori
  if !added.empty?
    puts "Added #{className}"
    puts added
    puts "\n\n"
  end

  return added, deleted, x_ori, x_mod if return_names
  return added, deleted
end
```

I didn't find any concerning differences.

Construction sets for eg are named per ASHRAE 169-2013 now versus ASHRAE 169-2006 before. 

```
 compare("DefaultConstructionSet")
Deleted DefaultConstructionSet:
90.1-2013 - Warehouse - ASHRAE 169-2006-8B
[... plenty more ...]

Added DefaultConstructionSet
90.1-2013 - Warehouse - ASHRAE 169-2013-7B
```

```
[42] diff(main)> compare("BoilerHotWater")
=> [[], []]
[43] diff(main)> compare("ChillerElectricEIR")
=> [[], []]
[44] diff(main)> compare("SpaceType")
Added SpaceType
SmallDataCenterLowITE ComputerRoom
Laboratory Office
SmallDataCenterHighITE ComputerRoom
Laboratory Equipment corridor
Laboratory Lab with fume hood
Laboratory Open lab
LargeDataCenterHighITE StandaloneDataCenter
LargeDataCenterLowITE StandaloneDataCenter
```

Diff all object names:

```
added, deleted, x_ori, x_mod = compare("ModelObject")
File.open('ori_names.txt', 'w') {|f| f.puts x_ori.sort.uniq}
File.open('mod_names.txt', 'w') {|f| f.puts x_mod.sort.uniq}
```

**Comment by jmarrec** [comment]
Generating a single template:

*** Finished all templates at: 2020-10-07 11:28:21 +0200, time elapsed = 15.5 min.

**Comment by jmarrec** [comment]
Ok, I ran all 16 templates after fixing the Parallel thing, took 36min. DEER 2017 failed though.

**Comment by jmarrec** [comment]
Re https://github.com/openstudiocoalition/OpenStudioApplication/pull/246#pullrequestreview-504407829

@macumber The "problem" with DEER 2017 is that the data doesn't contain the boilers.

```ruby
template_name = "DEER 2017"
std_applier = Standard.build(template_name)
std_applier.standards_data.keys
=> ["parking",
 "hvac_inference",
 "climate_zone_sets",
 "materials",
 "climate_zones",
 "fans",
 "constructions",
 "size_category",
 "schedules",
 "curves",
 "entryways",
 "chillers",
 "refrigeration_walkins",
 "refrigeration_system_lineup",
 "prototype_inputs",
 "construction_properties",
 "elevators",
 "refrigerated_cases",
 "unitary_acs",
 "space_types",
 "refrigeration_condenser",
 "exterior_lighting",
 "refrigeration_compressors",
 "motors",
 "heat_pumps",
 "refrigeration_system",
 "heat_rejection",
 "heat_pumps_heating",
 "water_heaters",
 "economizers",
 "construction_sets"]
```

File: developer/ruby/export_openstudio_standards_libraries.rb [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+
+      super(object, str, units)
+    end
+
+    def sqlFile
+      return FakeSqlFile.new
+    end
+  end
+end
+end
```
Comment (jmarrec) [inline developer/ruby/export_openstudio_standards_libraries.rb:43]: @macumber I've merged the branch since this works now. But I'm curious why you added that bit, and whether we couldn't at least remove the "OH YEAH" or something.

Comment (macumber) [reply developer/ruby/export_openstudio_standards_libraries.rb:43]: This code in Standards requires a sizing run so it can find sizes.  I wanted to fake that out so we didn't have to do a sizing run to generate standards OSMs.  I just left in OH YEAH so I could see where it was being called.  It's just for debugging, can be removed
